WebhookNotification/__tests__/handler.test.ts

```diff
@@ -126,16 +132,25 @@ const aCommonMessageData = {
   senderMetadata: aSenderMetadata
 };
 
+const aRetrievedProfileWithVerboseNotificationEnabled = {
```

Comment:
Why not `aRetrievedProfileWithFullPushNotificationsContentType`?

Reply:
Done in [60ac8fb](https://github.com/pagopa/io-functions-services/pull/232/commits/60ac8fb4f0dbb952acb844392008ef75b5bb4ace).